.travis.yml

```diff
@@ -37,6 +37,7 @@ env:
     - KONG_TEST_DATABASE=postgres TEST_SUITE=integration
     - KONG_TEST_DATABASE=cassandra CASSANDRA=2.2 TEST_SUITE=integration
     - KONG_TEST_DATABASE=cassandra CASSANDRA=3.9 TEST_SUITE=integration
+    - KONG_TEST_DATABASE=off TEST_SUITE=dbless
```

Comment:
Presumably db-less is faster than pg or C*? Maybe we could make Travis-ci to run db-less first, and if it passes, then run the other dbs, in a similar way to what it does with unit tests.

Reply:
The dbless suite is very incomplete as opposed to the others right now so it wouldn't really gain us a lot of insight. Also, splitting phases has an overhead in which it delays the startup of the next jobs, which is considerable. So mayyybe something to consider once the dbless suite is more complete, but I wouldn't do it right now.